ui/src/file_upload.tsx

```diff
@@ -326,9 +338,9 @@ export const
               )
               : (
                 <>
-                  {label && <Fluent.Label style={{ paddingTop: 6 }}>{label}</Fluent.Label>}
+                  {label && <Fluent.Label style={{ paddingTop: 6 }} required={required}>{label}</Fluent.Label>}
                   <div className={css.compact}>
-                    <Fluent.TextField data-test={`textfield-${name}`} readOnly value={fileNames} errorMessage={error} />
+                    <Fluent.TextField data-test={`textfield-${name}`} readOnly value={fileNames} errorMessage={error} required={label ? undefined : required} />
```

Comment:
What does `required` do in `Textfield`? Isn't including it on `Label` enough?

Reply:
@mturoci it is not enough, since `label` is just an optional prop. If label is not present, the asterisk shoud be displayed right to the input. The same behavior is implemented for other form components so I can't see any reason to make this component inconsistent with others. 
![image](https://user-images.githubusercontent.com/23740173/192226468-3b2a7753-c095-4ab5-a819-dd859d1ae30d.png)

Reply:
> The same behavior is implemented for other form components

Didn't know that! Agree then, let's leave it as is.